src/main/webapp/app/app.main.ts

```diff
@@ -3,6 +3,15 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { ProdConfig } from './core/config/prod.config';
 import { ArtemisAppModule } from './app.module';
 
+import * as Chart from 'chart.js';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+
+/**
+ * This plugin registers itself globally,
+ * meaning that once imported, all charts will display labels.
+ * In our case we want it enabled only for a few charts, so we first need to unregister it globally
+ */
+Chart.plugins.unregister(ChartDataLabels);
```

Comment:
this is really ugly :-(
Is there no other option to display labels?

Reply:
I guess we don't need this plugin at all.
Have a look at the quiz charts. They already include data labels. Example

<img width="843" alt="image" src="https://user-images.githubusercontent.com/744067/90399198-de74bf80-e09a-11ea-8083-d7d89fede727.png">